src/defaults/defaults.jl

```diff
@@ -221,14 +221,15 @@ end
 ##########################################################################################
 
 function get_building_polys(buildings)
-    building_polys = fill(Point2f[], length(buildings));
-    
+    building_polys = GeometryBasics.Polygon[]
+
     for (i, (id, b)) in enumerate(buildings)
```

Comment:
This isn't needed anymore. we can just do

```suggestion
    for b in values(buildings)
```